### PR TITLE
Docs: Add log topic libiresearch. Remove compactor, datafiles

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of #15629

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md [hunk 1]
```diff
@@ -214,172 +214,169 @@ API is enabled, accessing it requires admin privileges, or even superuser
...
 @RESTBODYPARAM{clustercomm,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{collector,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{communication,string,optional,string}
-One of the possible log levels.
```
Comment (cpjulia) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:277]: should this not be replaced?

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md [hunk 2]
```diff
@@ -214,172 +214,169 @@ API is enabled, accessing it requires admin privileges, or even superuser
...
-
-@RESTBODYPARAM{compactor,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{config,string,optional,string}
-One of the possible log levels.
+One of the possible log topics.
 
 @RESTBODYPARAM{crash,string,optional,string}
```
Comment (cpjulia) [inline Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:282]: and this topic as well?

**Review by cpjulia — APPROVED** [review]
LGTM

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md (hunk 1 above)
Comment (Simran-B) [reply Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:277]: You mean log level > log topic? It is, see the added (green) line 277. It's a little obfuscated by the removal of the compactor topic.

File: Documentation/DocuBlocks/Rest/Administration/get_admin_log.md (hunk 2 above)
Comment (Simran-B) [reply Documentation/DocuBlocks/Rest/Administration/get_admin_log.md:282]: Same, level is replaced by topic but it's a little obfuscated by the removal of the datafiles topic.

**Comment by KVS85** [comment]
- [ ] https://jenkins.arangodb.biz:3456/view/Documentation/job/arangodb-ANY-examples/1718/

**Pull request merged**